client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientClusterFailoverEventsTest.java

```diff
@@ -0,0 +1,85 @@
+package org.infinispan.client.hotrod.event;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.annotation.ClientListener;
+import org.infinispan.client.hotrod.test.HotRodClientTestingUtil;
+import org.infinispan.client.hotrod.test.MultiHotRodServersTest;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.server.hotrod.HotRodServer;
+import org.testng.annotations.Test;
+
+import static org.infinispan.client.hotrod.test.HotRodClientTestingUtil.findServerAndKill;
+import static org.infinispan.client.hotrod.test.HotRodClientTestingUtil.killRemoteCacheManager;
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+
+@Test(groups = "functional", testName = "client.hotrod.event.ClientClusterFailoverEventsTest")
+public class ClientClusterFailoverEventsTest extends MultiHotRodServersTest {
+
+   @Override
+   protected void createCacheManagers() throws Throwable {
+      // Empty
+   }
+
+   public void testEventReplayWithAndWithoutStateAfterFailover() {
+      ConfigurationBuilder base = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false);
+      //base.clustering().hash().numOwners(1);
```

Comment:
Yeah, all I wanted to verify is that event data is replayed after failover, so making all data available in all nodes simplified the test.